content/editor.html

```diff
@@ -12,6 +12,8 @@
 </style>
 <link rel="stylesheet" type="text/css" href="//<!--#echo var="site"-->/editor.css">
 <link rel="stylesheet" type="text/css" href="//<!--#echo var="site"-->/lib/droplet.css">
+<link rel="stylesheet" href="https://code.jquery.com/ui/1.10.4/themes/flick/jquery-ui.css">
```

Comment:
Is there a reason this goes to another site, and the rest seem to be copied here?